src/utils/auth-utils.js

```diff
@@ -1,9 +1,9 @@
 const { config } = require("@config/config")
 
-const NODE_ENV = config.get("env")
+const NODE_ENV = config.get("env").toLowerCase()
 
 function isSecure() {
-  return NODE_ENV !== "DEV" && NODE_ENV !== "LOCAL_DEV" && NODE_ENV !== "test"
+  return NODE_ENV !== "dev" && NODE_ENV !== "staging" && NODE_ENV !== "test"
 }
```

Comment:
just realised this is the same as `IS_PROD_ENV`; can centralise but feel free to ignore (technically out of scope n small change)

Reply:
Hmm actually the only place this is being used is for our github auth redirect cookie settings - any idea why it's currently only set to secure on production and not staging? I was thinking it might be the same as the one we use in `server.js` instead to separate local dev/testing from staging/prod, we can probably refactor out the env checks if so

Reply:
i can't remember why it's not secure on `staging` tbh but it can't hurt HAHA